sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -729,6 +758,102 @@ public partial class LexicalAnalyzer
         public static implicit operator Azure.Search.Documents.Indexes.Models.LexicalAnalyzerName (string value) { throw null; }
         public static bool operator !=(Azure.Search.Documents.Indexes.Models.LexicalAnalyzerName left, Azure.Search.Documents.Indexes.Models.LexicalAnalyzerName right) { throw null; }
         public override string ToString() { throw null; }
+        public static partial class AsString
```

Comment:
Those are good alternatives. I only choose `AsString` as a carry-over from track 1. @KrzysztofCwalina, what do you prefer here? I'm liking `Names` over the others.

Reply:
Per arch meeting, we'll go with `Names`. I'll write a test to make sure the names here exactly match the names on the extensible enum.

Reply:
Sounds like we wanted `Names`.

Reply:
As I made this change, I think we should actually use `Values` since the constants are defined as the values. I started a thread internally and will change accordingly (for now, I've gone with `Names`).

Reply:
As discussed internally, we'll go with `Values`

/cc @pakrym @tg-msft @KrzysztofCwalina